artifact-authors.md

```diff
@@ -0,0 +1,224 @@
+- **`layerType`** - a value representing the short name of the type.
+- **`optional-layerSubType`** - provides additional extensibility of a `layerType`
+- **`version`** - provides artifact authors the ability to revision their schemas and formatting, enabling tools to identify which format they will process. Using a version enables artifact tools to identify the format for processing.
+- **`fileFormat`** - provides for standard, or custom formats, enabling artifact authors to leverage existing or custom content processing libraries.
+- **`optional-compressionFormat`** - in addition to the fileFormat, optional compression is used. (eg: `+gzip`, `+zstd`)
+
+### Example Layer Types
+
+| Artifact Type | mediaType |
+| - | - |
+| Simple text file layer | `application/text` |
+| Non-compressed OCI Image layer | `application/vnd.oci.image.layer.v1.tar` |
+| [gzip] Compressed OCI Image layer | `application/vnd.oci.image.layer.v1.tar+gzip` |
+| [gzip] Compressed Docker Image layer | `application/vnd.docker.image.rootfs.diff.tar+gzip` |
+| Non-compressed Helm Chart layer | `application/vnd.cncf.helm.chart.layer.v1.tar` |
+| [gzip] Compressed Helm Chart layer | `application/vnd.cncf.helm.chart.layer.v1.tar+gzip` |
+| Singularity SIF layer | `application/vnd.sylabs.sif.layer.v1.sif` |
+| Sample Doc Artifact Summary layer | `application/vnd.oci.sample-repo-doc.summary.layer.v1.md` |
+| Sample Doc Artifact Content layer| `application/vnd.oci.sample-repo-doc.content.layer.v1.tar` |
+
+## Optional: Defining Config Schema
+
+When defining an artifact type, the persistence of the artifact may be broken up into content and configuration. Configuration may be stored as a layer, or it may be stored and referenced by the `manifest.config`.
+
+While the value of `manifest.config.mediaType` is used to determine the artifact type, the persistance of a `config` object is OPTIONAL. Artifacts can push a null reference for `config.json` persistance.
+
+Some benefits of placing content in `manifest.config` include:
+
+- Tooling can pull the configuration object prior to layers. An artifact component may use the config to determine how and where the layer should be instantiated. The artifact component might send layer request to different compute instances; such as OCI Image layers being sent to a Windows or Linux Hosts.
+- Registries may opt-into parsing the configuration if it provides meaningful top-level information. For example, [OCI Image Config][image-spec-config] stores `OS`, `Architecture` and `Platform` information that some registry operators may wish to display. The config is easy to pull & parse, as opposed to getting a layer url to pull, possibly decompress and parse.
+
+Registries which implement the artifact spec MAY:
+
+- Parse and process the contents of  `manifest.config`, based on the provided schema of `manifest.config.mediaType`, offering additional information or actions.
+- Ignore the contents and validation of the `config` object.
```

Comment:
I'd be clear here: "Ignore the contents and validation of the referenced `config` data`". The "config object" could refer to the config object inside the manifest, which registries do need to validate (since it contains the mimetype)

Reply:
I believe I captured the feedback relative to this artifact authors doc. There's a lot of great content we'll add to the `registry-operator-how-to.md` PR coming later.